src/Sarif/Writers/SarifLogger.cs

```diff
@@ -104,12 +104,12 @@ public class SarifLogger : BaseLogger, IDisposable, IAnalysisLogger
                 dataToInsert.HasFlag(OptionallyEmittedData.ContextRegionSnippets))
             {
                 _insertOptionalDataVisitor = new InsertOptionalDataVisitor(dataToInsert,
+                                                                           fileRegionsCache,
```

Comment:
>fileRegionsCache [](http://example.com/codeflow?start=75&length=16)

this is a required argument so had to be moved in the parameters list. This is therefore documented as a breaking change in the SDK.